package.json

```diff
@@ -15,11 +15,11 @@
     "dotenv": "^16.4.7",
     "hono": "^4.0.9",
     "openai": "^4.79.3",
-    "spinai": "*",
-    "tsx": "^4.7.1"
+    "spinai": "*"
   },
   "devDependencies": {
     "@types/node": "^20",
+    "tsx": "^4.19.2",
```

Comment:
also no newline at EOF? cmonnn, you're better than that 😭 add it to keep it clean.